src/test-helpers.tsx

```diff
@@ -174,25 +175,36 @@ export const createVersionWithEntries = (
   return createInternalVersion(createExternalVersionWithEntries(...args));
 };
 
+export const createVersionAndEntryStatusMap = (
+  ...args: Parameters<typeof createExternalVersionWithEntries>
+) => {
+  return {
+    version: createVersionWithEntries(...args),
+    entryStatusMap: createEntryStatusMap(
+      createExternalVersionWithEntries(...args),
+    ),
+  };
+};
+
 export const getFakeVersionAndPathList = (
   entries: ({ path: string } & Partial<ExternalVersionEntry>)[],
 ) => {
   const pathList = entries.map((e) => e.path);
 
-  const version = createVersionWithEntries(
-    entries.map((params) =>
-      createInternalVersionEntry({
-        ...fakeVersionEntry,
-        mime_category: 'text',
-        status: 'M',
-        ...params,
-        filename: params.path,
-      }),
-    ),
+  const externalEntries: ExternalVersionEntry[] = entries.map((params) => ({
+    ...fakeVersionEntry,
+    mime_category: 'text',
+    ...params,
+    filename: params.path,
+  }));
+
+  const { version, entryStatusMap } = createVersionAndEntryStatusMap(
+    externalEntries,
   );
+
   version.selectedPath = pathList[0];
 
-  return { pathList, version };
+  return { entryStatusMap, pathList, version };
```

Comment:
> I did not update the function name yet, since it seems too verbose to call it `getFakeVersionEntryStatusMapAndPathList`.....

I agree. The name is fine as is. I guess you could call it `getFakeVersionAndPathData` but it's not too important.